### PR TITLE
Add TLS configurations to Kafka WS and Kong

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
New features in Kafka WS and Kong:
- Add volumes for cecrtificates
- Add configurations for TLS communication
It also corrects minor typos, updates the variables docs and removes the old EJBCA files.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: inventories/example_local/group_vars/dojot-k8s/dojot.yaml [hunk 1]
```diff
@@ -35,12 +39,21 @@ dojot_vernemq_replicas: 1
...
+
 # Enables fixed nodeports
 dojot_fixed_nodeports_enabled: false
 
 # Fixed nodeports to use. It's used if dojot_fixed_nodeports_enabled is true
 dojot_nodeports:
-  apigw: 30001
+  apigw:
+    http: 30001
+    https: 30002
```
Comment (andersonluisribeiro) [inline inventories/example_local/group_vars/dojot-k8s/dojot.yaml:56]: Should this port be mapped in haproxy?

Comment (eduardogmisiuk) [reply inventories/example_local/group_vars/dojot-k8s/dojot.yaml:56]: Yes, it should be mapped to HTTPS default port (443).

Comment (andersonluisribeiro) [reply inventories/example_local/group_vars/dojot-k8s/dojot.yaml:54]: Ok, won't the mapping be treated in this PR?

Comment (eduardogmisiuk) [reply inventories/example_local/group_vars/dojot-k8s/dojot.yaml:54]: Yes, it will be treated, I didn't see this configuration.

**Pull request merged**